db/db.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+mongoose.connect('mongodb://localhost/fec', {useNewUrlParser: true });
+
+const reviewSchema = new mongoose.Schema({
+  uId: Number,
+  bId: Number,
+  rating: Number,
+  reviewText: String,
+  checkin: Boolean,
+  useful: [], // Contains the users([Username,UserID]) who thought review was useful
```

Comment:
Even though I didn't define a specific type in the array as well..lol But I plan to modify this part when I have chance to go back...It is always good to have defined data type in the schema for data validation.

Reply:
I agree, I initially wanted to use the word Array instead of using an actual [], but I couldn't find any examples of someone using 'Array' online.